fixture-monkey-datafaker/src/main/java/com/navercorp/fixturemonkey/datafaker/property/DataFakerProperty.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Fixture Monkey
+ *
+ * Copyright (c) 2021-present NAVER Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.navercorp.fixturemonkey.datafaker.property
+
+import com.navercorp.fixturemonkey.api.property.Property
+
+class DataFakerProperty(private val originalProperty: Property) : Property {
```

Comment:
Could it be replaced with 'originalProperty'?

Reply:
I think there is an error in the translation.ㅋㅋㅋㅋㅋ

I mean using the keyword 'by' for delegation.
https://kotlinlang.org/docs/delegation.html#overriding-a-member-of-an-interface-implemented-by-delegation